licenses.d2iq.yaml

```
@@ -82,11 +82,11 @@ resources:
       - license_path: LICENSE.txt
         ref: ${image_tag}
         url: https://github.com/jimmidyson/configmap-reload
-  - container_image: ghcr.io/mesosphere/dkp-container-images/docker.io/kiwigrid/k8s-sidecar:1.25.3-d2iq.1
+  - container_image: docker.io/kiwigrid/k8s-sidecar:1.27.6
     sources:
       - license_path: LICENSE
-        ref: ${image_tag%-d2iq.1}
-        url: https://github.com/kiwigrid/k8s-sidecar
+        ref: ${image_tag}
+        url: docker.io/kiwigrid/k8s-sidecar
```

```suggestion
        url: https://github.com/kiwigrid/k8s-sidecar
```

The URL is a location of git repository.